#// Структура отправляемых данных от Data к контроллеру Driver

	uint32 id                   # Номер команды по порядку
	SControl control      # Структура управления машиной
	SLed led              # Управление светодиодами
	uint32 cheksum              # Контрольная сумма данных в структуре